.circleci/config.yml

```diff
@@ -2368,7 +2392,7 @@ jobs:
             cd /swirlds-platform;
             sed -i -e 's/github.com/services-jrs-regression/g' .gitmodules;
             git submodule update --init --recursive --checkout;
-            cd regression;
+            cd regression; git checkout fix-15-node-tests;
```

Comment:
Should revert before merging

Reply:
Oh, nvm, didn't notice that this PR was closed.

Reply:
Hi @qnswirlds , I have closed this Pr and opened another one https://github.com/hashgraph/hedera-services/pull/1041 as this was not syncing the changesI pushed after creating the PR. Thanks. Could you please review the new PR